### DISCUSSION TITLE
App store [Aug 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6161] [Discord Threads]
> 🙋‍♂️ **productivix**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### App store is now a global menu. You can find details here   
#### https://docs.nocodb.com/setup-and-usages/account-settings#app-store  
#  
> 🙋‍♂️ **kronk0s**  
#### I have the same issue. I looked in the global menu and the App store wasn't there  
#  
> 👨‍🦲 **dstala**  
#### only super-admin will be able to access. are you on 'super admin' previleges?  
#  
> 🙋‍♂️ **kronk0s**  
#### yes I'm the superadmin  
#  
> 🙋‍♂️ **productivix**  
#### haaaaa.. I understand , and share what solves all login issues : I expain : there is 2 levels for the admin  : (difficult to catch and access I must say: the global level and the project level : see the screnn copies , and in the global you can solve the user login password lost , the smtp email install , the store, etc ... To access the global menu, click on the "N" only on top left of the project menu and then on account settings on top right , tricky way but exists . Thanks <@857998597474484295>  
![image0](https://cdn.discordapp.com/attachments/1124302598556631161/1126068669571682397/Capture_decran_globalmenu.png)  
![image1](https://cdn.discordapp.com/attachments/1124302598556631161/1126068669777186867/Capture_decran_project_menu.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1124302598556631161)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
